src/main/java/com/gooddata/warehouse/Warehouse.java

```diff
@@ -75,7 +77,8 @@ public Warehouse(@JsonProperty("title") String title, @JsonProperty("authorizati
                      @JsonProperty("createdBy") String createdBy, @JsonProperty("updatedBy") String updatedBy,
                      @JsonProperty("status") String status, @JsonProperty("environment") String environment,
                      @JsonProperty("connectionUrl") String connectionUrl,
-                     @JsonProperty("links") Map<String, String> links) {
+                     @JsonProperty("links") Map<String, String> links,
+                     @JsonProperty("license") License license) {
```

Comment:
moreover you probably haven't read https://github.com/gooddata/gooddata-java/blob/master/CONTRIBUTING.md#enums